src/InstrumentationEngine/InstrumentationEngine.vcxproj

```diff
@@ -148,7 +148,7 @@
     <ClCompile Include="TokenType.cpp" />
     <ClCompile Include="Type.cpp" />
     <ClCompile Include="TypeCreator.cpp" />
-    <ClInclude Include="ProfilerManagerForInstrumentationMethod.h" />
+    <ClInclude Include="*.h" />
```

Comment:
Intended?

Reply:
Yep, it originally was *.h, and when I added my ProfilerManagerForInstrumentationMethod.h file in VS it overwrote this so VS stopped displaying all of the other .h files. 

---
In reply to: [405691979](https://github.com/microsoft/CLRInstrumentationEngine/pull/231#discussion_r405691979) [](ancestors = 405691979)

Reply:
MSBuild is now generating a warning about this, I'll revert *.h to explicit files.